src/components/list-podcast/script.js

```diff
@@ -19,7 +19,7 @@ export default {
       this.$emit('add-favorite', trackId);
     },
     playTrack(trackId) {
-      this.$emit('play-track', trackId);
+      this.$root.$emit('play-track', trackId);
```

Comment:
Just remember to setup this as a constant

Reply:
Done.

Reply:
Done.